example/lstm_master.py

```diff
@@ -1,6 +1,9 @@
 import numpy
 import time
 
+import sys
+sys.path.append('..')
```

Comment:
Since `..` is relative to the current directory, not the directory containing `lstm_master.py`, adding things to the path can lead to strange behaviours if the user has Python files in the parent directory.
I think we could do one of:
- put `__file__/..` in sys.path instead
- make an actual Python package for platoon, so we can import ..channel from the example
- update the doc to make sure channel.py's parent dir is in the PYTHONPATH